normandy/recipes/models.py

```diff
@@ -121,16 +150,40 @@ def __repr__(self):
     def __str__(self):
         return self.name
 
-    def save(self, *args, **kwargs):
+    def save(self, *args, skip_last_updated=False, **kwargs):
         # Increment the revision ID and save the reversion if we've
         # changed
         if self.is_dirty(check_relationship=True):
+            dirty_fields = self.get_dirty_fields(check_relationship=True)
+
+            # Any change will invalid the signature
+            dirty_field_names = list(dirty_fields.keys())
+            if dirty_field_names != ['signature'] and self.signature is not None:
+                self.signature.delete()
+
+            # Check for changes that should disable the recipe
```

Comment:
Hmmmmm... so this got removed when we removed the peer approval changes from master. The `EDITABLE_FIELDS_WHITELIST` property got left in (which it shouldn't have been).

I think we should not include this in this PR.